genesis/src/lib.rs

```diff
@@ -20,12 +21,25 @@ use iroha_data_model::{
 use once_cell::sync::Lazy;
 use serde::{Deserialize, Serialize};
 
-/// [`DomainId`] of the genesis account.
-pub static GENESIS_DOMAIN_ID: Lazy<DomainId> = Lazy::new(|| "genesis".parse().expect("Valid"));
-
-/// [`AccountId`] of the genesis account.
-pub static GENESIS_ACCOUNT_ID: Lazy<AccountId> =
-    Lazy::new(|| AccountId::new(GENESIS_DOMAIN_ID.clone(), "genesis".parse().expect("Valid")));
+/// [`DomainId`](iroha_data_model::domain::DomainId) of the genesis account.
+pub static GENESIS_DOMAIN_ID: Lazy<DomainId> = Lazy::new(|| "genesis".parse().unwrap());
+
+/// [`AccountId`](iroha_data_model::account::AccountId) of the genesis account.
+pub static GENESIS_ACCOUNT_ID: Lazy<AccountId> = Lazy::new(|| {
+    AccountId::new(
+        GENESIS_DOMAIN_ID.clone(),
+        GENESIS_ACCOUNT_KEYPAIR.public_key().clone(),
+    )
+});
+
+/// [`KeyPair`] of the genesis account.
+pub static GENESIS_ACCOUNT_KEYPAIR: Lazy<KeyPair> = Lazy::new(|| {
+    KeyPair::new(
+        iroha_crypto::PublicKey::from_str("ed0120E2ECD69DA5833EC10FB3DFAED83A07E5B9CBE9BC39484F0F7DDEC8B46253428B").unwrap(),
+        iroha_crypto::PrivateKey::from_hex(iroha_crypto::Algorithm::Ed25519, "DD61D7A2244A504E78BA80383DFCC0228E25CA131E5A6AF503F71632D23BD76AE2ECD69DA5833EC10FB3DFAED83A07E5B9CBE9BC39484F0F7DDEC8B46253428B").unwrap(),
+    )
+    .unwrap()
+});
```

Comment:
Is it right that Iroha has a hardcoded magic key pair associated with the genesis account?

I thought that the intention is that Iroha should not know genesis private key at all, and receive the public one via configuration (alongside with pre-signed genesis transaction).

Reply:
It is not intended to reflect even ongoing changes. I'd keep refactoring to a minimum where it is outside of the main focus

Reply:
Ok, GENESIS_ACCOUNT_ID would be obtained from the configuration. Then GENESIS_ACCOUNT_KEYPAIR would be removed